src/components/fileUpload.js

```diff
@@ -322,12 +498,129 @@
           ],
         },
       },
-      fileList: {
+      listView: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'space-between',
+      },
+      gridView: {
+        display: 'flex',
+      },
+      gridItem: {
+        display: 'flex',
+        borderRadius: '0.3rem',
+        flexDirection: 'column',
+        border: '1px solid #eee',
+        marginRight: '0.9375rem',
+        marginBottom: '0.9375rem',
+      },
+      gridItemDetails: {
+        maxWidth: ({ options: { imagePreviewWidth, showImagePreview } }) =>
+          showImagePreview ? imagePreviewWidth : 'auto',
+        display: 'flex',
+        margin: '0.9375rem',
+        justifyContent: 'space-between',
+      },
+      devImage: {
+        display: 'flex',
+        overflow: 'hidden',
+        margin: '0.9375rem',
+        borderStyle: 'dashed',
+        borderColor: '#AFB5C8',
+        borderRadius: '0.3rem',
+        borderWidth: '0.0625rem',
+        border: '2px dashed black',
+        backgroundColor: '#F0F1F5',
+        width: ({ options: { imagePreviewWidth } }) => imagePreviewWidth,
+        height: ({ options: { imagePreviewHeight } }) => imagePreviewHeight,
+      },
+      gridDevImage: {
+        extend: 'devImage',
+
+        margin: 0,
+        borderRadius: '0.3rem 0.3rem 0 0',
+        width: ({ options: { imagePreviewWidth } }) => imagePreviewWidth,
+        height: ({ options: { imagePreviewHeight } }) => imagePreviewHeight,
+      },
+      deleteIcon: {
+        color: `${t.colors.light}!important`,
```

Comment:
Can you apply the same consistency here as I asked in a previous comment with the margins? Either string interpolate or use the array notation on all of them.